events-api/src/mbus_nats.rs

```diff
@@ -320,7 +318,12 @@ impl<T: Serialize + DeserializeOwned> BusSubscription<T> {
                 let message = match message {
                     Ok(message) => message,
                     Err(error) => {
-                        tracing::warn!(%error, "Error accessing jetstream message");
+                        match error.kind() {
+                            MessagesErrorKind::ConsumerDeleted => (), // TODO create consumer again
+                            MessagesErrorKind::MissingHeartbeat => (), /* TODO check if stream */
+                            // and consumer exists,
+                            _ => tracing::warn!(%error, "Error accessing jetstream message"),
+                        };
```

Comment:
Don't mix comment types.

```suggestion
                        match error.kind() {
                            // TODO: create consumer again.
                            MessagesErrorKind::ConsumerDeleted => (), 
                            // TODO: check if stream and consumer exists.
                            MessagesErrorKind::MissingHeartbeat => (),
                            _ => tracing::warn!(%error, "Error accessing jetstream message"),
                        };
```